packages/next/src/build/compiler.ts

```diff
@@ -58,34 +59,37 @@ export function runCompiler(
     }
     compiler.fsStartTime = Date.now()
     compiler.run((err, stats) => {
+      const compilerResult = runWebpackSpan
+        .traceChild('webpack-generate-error-stats')
+        .traceFn(() =>
+          generateStats({ errors: [], warnings: [], stats }, stats!)
```

Comment:
why change this,  it seems these changes is just reorder the code, I check the origin code and it seems works find in rspack